backend/utils/date_helper.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
```

Comment:
What about combining some of the utils files into one. perhaps `helpers.py` or something could include date helper and redis helper

Reply:
I recombined these for now.  If it gets too big we can split out function later.